python/sglang/srt/layers/quantization/__init__.py

```diff
@@ -112,7 +112,15 @@ def get_quantization_config(quantization: str) -> Type[QuantizationConfig]:
             f"Invalid quantization method: {quantization}. "
             f"Available methods: {list(QUANTIZATION_METHODS.keys())}"
         )
```

Comment:
```suggestion
        )
        if is_cpu():
            return QUANTIZATION_METHODS[quantization]
```

we should also have a list e.g. `CPU_QUANTIZATIPON_METHODS` something like this, and do checks here.

Reply:
yes, created `CPU_QUANTIZATIPON_METHODS `.